python-meh.spec

```diff
@@ -1,10 +1,8 @@
-%{!?python_sitelib: %define python_sitelib %(%{__python} -c "from distutils.sysconfig import get_python_lib; print get_python_lib()")}
-%global with_python3 1
-
-%define libreportver 2.0.18-1
+%global libreportver 2.0.18-1
+%global srcname meh
```

Comment:
this doesn't make the specfile any more readable. consider not adding unnecessary complexity for zero gain. it's not like I can just copy paste the entire specfile and change the definition here to `hah` and tadah, or is it?

Reply:
Good point, fixed.